os/board/rtl8721csm/src/component/common/api/wifi/wifi_conf.c

```diff
@@ -2827,4 +2831,112 @@ int wifi_set_null1_param(uint8_t check_period, uint8_t limit, uint8_t interval)
 }
 #endif
 
+#include "freertos/wrapper.h"
+extern Rltk_wlan_t rltk_wlan_info[NET_IF_NUM];
+
+typedef struct WIFI_MONITOR{
+	/*RX related*/
+	unsigned long rx_success_pkt_count;
+	unsigned long rx_success_bytes_count;
+	unsigned long rx_dropped_pkt_count;	
+	int rx_rssi;
+	u32 rxd_beacon_count;	
+	
+	/*TX related*/
+	unsigned long tx_success_pkt_count;
+	unsigned long tx_success_bytes_count;	
+	unsigned long tx_dropped_pkt_count;
+}wifi_monitor;
+
+wifi_monitor wifi_monitor_info = {0}; 
+wifi_monitor wifi_monitor_prev = {0};
+
+void wifi_get_monitor_info(unsigned char idx)

```

Comment:
idx seems that always 0, actually this is called by _wifi_info_timer_handler only.
Hence we could remove parameter.

Reply:
Done.